tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/session/FileSessionDistributionTests.java

```diff
@@ -16,9 +16,7 @@
 import java.util.Collections;
 import java.util.List;
 
-/**
- *
- */
+@Testcontainers(disabledWithoutDocker = false)
```

Comment:
Oddly, this test doesn't fail on Windows, and it doesn't have docker.

Reply:
yup because this test is not using docker :)  it's only session storage in files